Automationexercise/PageObjects/Cart.cs

```diff
@@ -0,0 +1,57 @@
+using AutomationExercise;
+using Microsoft.Playwright;
+
+namespace Automationexercise.PageObjects;
+
+public class Cart (IPage page, string baseUrl) : BasePage(page: page, baseUrl)
+{
+    //Selectors & Locators
+    public ILocator GoShoppingButton => Page.Locator("#cartModal .btn-success");
+    private ILocator ViewCartLink => Page.Locator("#cartModal [href='/view_cart']");
+    private ILocator ProceedToCheckoutButton => Page.Locator(".btn.check_out");
+
+    //Methods
+    public async Task AddToCart(string productName)
+    {
+        if (productName == null) throw new ArgumentNullException(nameof(productName));
+        var cartButton = Page.Locator($"//div[@class=\"features_items\"]//div[contains(@class,'productinfo ')]/p[contains(text(),'{productName}')]");
```

Comment:
instead of \" you can use just '

Reply:
fixed. For some reason, some xpath helpers (browser extension) don't work if the class name in the ' '